taxi/views.py

```diff
@@ -1,3 +1,16 @@
+from django.http import HttpRequest, HttpResponse
 from django.shortcuts import render
 
-# Create your views here.
+from taxi.models import Car, Driver, Manufacturer
+
+
+def index(request: HttpRequest) -> HttpResponse:
+    num_drivers = Driver.objects.count()
+    num_manufacturers = Manufacturer.objects.all().count()
+    num_cars = Car.objects.all().count()
```

Comment:
.all() is redundant
When you run all, django fetches all records and then counts them
Instead you can make it without all() so that only 1 sql query will be called